pkg/nodebootstrap/ubuntu_test.go

```diff
@@ -49,7 +49,8 @@ var _ = Describe("Ubuntu User Data", func() {
 API_SERVER_URL=
 B64_CLUSTER_CA=
 NODE_LABELS=
-NODE_TAINTS=`, "\n")))
+NODE_TAINTS=
+CONTAINER_RUNTIME=`, "\n")))
```

Comment:
this flag isn't for ubuntu, granted the var wont be used, but could it be confusing if a user comes across it? maybe have `makeBootstrapEnv` only add the var if al2? wdyt?

Reply:
Ah, agreed. It is confusing.